### DISCUSSION TITLE
Setiing up Torrent client with sablier

### DISCUSSION THREAD

**Discussion by TheRedCyclops** [sablierapp/sablier#431] [Q&A]
Hi, I want to setup qBittorrent with sablier, but I'm pretty sure that the way that sablier detects traffic is though the reverse proxy and currently the traffic from qbt is going directly out to the internet, how would I go about configuring either sablier or the reverse proxy (traefik) to properly handle this and not shutdown the client in the middle of a download?

**Comment by acouvreur** [comment]
Hello @TheRedCyclops 

Sablier won't detect that activity, although it could to keep the container alive.

But Sablier is not fit for background running applications such as torrent clients which should run all the time.

You can still use it, but as you mentioned, currently it may stop the container while it's downloading.
